fendermint/app/src/options/key.rs

```diff
@@ -21,6 +23,19 @@ pub struct KeyArgs {
     pub command: KeyCommands,
 }
 
+#[derive(Args, Debug)]
+pub struct AddPeer {
+    /// The path to node key file.
+    #[arg(long, short)]
+    pub node_key_file: PathBuf,
+    /// The path to a local file with persistent peers.
```

Comment:
It would be good to explain a little bit more whether this is some standard CometBFT file that people should find, or an arbitrary path where the peer IDs will be added (I think this is the case), so that they can just put down whatever they want. Maybe say it will be created if it doesn't exist.

Reply:
Fixed.